src/ansys/aedt/core/modeler/circuits/primitives_maxwell_circuit.py

```diff
@@ -284,3 +284,87 @@ def create_winding(self, name=None, location=None, angle=0, use_instance_id_netl
         )
         id.set_property("Name", name)
         return id
+
+    @pyaedt_function_handler(compname="name")
```

Comment:
@DaveTwyman why are you using the function handler? you are not using compname. This was used by us to deprecate old arguments not following python conventions. This is a copy/paste error I guess.

Reply:
@gmalinve , I'm using the function handler because the other five components in the Maxwell library use it so I've followed the same structure.

Reply:
@DaveTwyman sorry I may have typed it wrongly. You have to use the function handler but why do you need compname?

Reply:
@SMoraisAnsys ,@gmalinve. Got it, I'll delete this from the two new components.

Reply:
Marked as resolved, I've changed 'id' to 'comp_name' throughout 'primitives_maxwell_circuit'

Reply:
Please push your changes before marking as resolved. Currently I'm still seeing the same behavior as the one we mentionned.

Reply:
Done, Thanks